bin/client/main.rs

```diff
@@ -145,7 +145,7 @@ async fn run() -> std::result::Result<(), ClientError> {
         .build(&opts.server_url, token, keyring)?;
 
     let local = bindle::provider::file::FileProvider::new(
-        bindle_dir,
+        bindle_dir.clone(),
```

Comment:
Nit: I think you can just pass a borrow here actually

Reply:
fixed